### ISSUE TITLE
Is it possible to make it so that when typing a numbered list it does not auto-increment?

### ISSUE THREAD

**Issue by skleanthous** [RickStrahl/MarkdownMonster#356]
When I write a numbered list, I want all numbers to be 1. The reason is that very oftern in editing (later or even during first draft) I go and add a number midway in the list. This results in lists that have odd\repeated numbers at random.

I would really like the ability to turn this off in settings.

An example of what I end up with after edits:
``` md
1. asd
2. asd
3. asd
4. asd
3. asd
4. asd
5. asd
6. asd
7. asd
8. asd
5. asd
```

I would far more prefer for all of these to be 1.

If this feature is OK with maintainers, I can create a PR.

**Comment by RickStrahl** [comment]
That sounds like a very odd request to me. Why would you ever do this?

It's not supported by Markdown or HTML so not going to happen with standard syntax.

You can do what you want with raw HTML though:

```markdown
<div>1. asd</div>
<div>1. asd</div>
<div>1. asd</div>
<div>1. asd</div>
```

or you can add custom styling to use lists with a custom UL tag, but either way, markdown doesn't support this.

**Comment by skleanthous** [comment]
Hello @RickStrahl ,

I am sure you misunderstood me :) I don't want the compiled result to render all in 1, I want in the _editor_ to be able to turn off auto-increment when I press enter after I type the line. Please see the example I pasted before (this is from the editor):
```
1. asd
2. asd
3. asd
4. asd
3. asd
4. asd
5. asd
6. asd
7. asd
8. asd
5. asd
```

**Comment by RickStrahl** [comment]
I don't disagree with you about the list behavior. I would much prefer ACE didn't do anything with list items and just let you type as is without any fix ups, but alas... 
You can type whatever number you want into the editor. In fact I type my number lists like this:

```markdown
1. Item 1
1. Item 2
1. Item 3
```

So that I don't have to worry about the numbering. The Markdown parser will fix these up properly when rendered. I find this easier than dealing with the out of sync numbering that you eventually end up with.

What could be done on my is to have a 'reformat' option on the context menu that can detect you're inside of a numbered list and offer to renumber it for you.

As to fixing the editor behavior - as I've mentioned on other threads, we're using ACE Editor for the core editing features and they provide the Markdown parsing. It's extremely tricky fixing the editor behaviors - I've tried my hand at it a few times and just managed to muck things up...  There is at least one  'Help Wanted' open still open regarding lists. If you really want to see different behavior the best place to ask is in the ACE repository. We merge ACE back regularly.

* [ACE Editor Issues](https://github.com/ajaxorg/ace/issues)
* [ACE Editor Kitchen Sink Sample](https://ace.c9.io/build/kitchen-sink.html)

**Comment by skleanthous** [comment]
Hey @RickStrahl, yes this is what I meant and this is what I'm doing too (writing all my lists as 1. etc), but it is a bit of a problem when I brainstorm (it helps me a lot to type ideas when I brainstorm) and I go back to "fix" it. I'll try and give the ACE issue a go, but js is not my strong suit :)

Also: I haven't been using Markdown monster for a long time, but its awesome so far. Thanks.

**Comment by RickStrahl** [comment]
You can now turn of the auto-completion - see the other thread. Autocomplete is now disable by default and enabled via `Editor.EnableBulletAutoCompletion`. 

It's available in 1.10.16.1 and later.
